src/domain/accounts/index.types.d.ts

```diff
@@ -77,7 +77,7 @@ type Account = {
   readonly isEditor: boolean
   readonly quizQuestions: UserQuizQuestion[]
   kratosUserId: UserId
-
+  displayCurrency: string
```

Comment:
we could make a branded type for it? 

```suggestion
  displayCurrency: DisplayCurrency
```

Reply:
currently we have a DisplayCurrency type (enum), we will replace it in the next PR because we cant use the same because currencies are dynamic